west.yml

```diff
@@ -79,6 +79,12 @@ manifest:
       path: modules/hal/gigadevice
       groups:
         - hal
+    - name: hal_hisilicon
```

Comment:
Please note that you need to create an issue with a new HAL request. Otherwise this won't be pulled.